src/containers/ClusterInfo/ClusterInfo.tsx

```diff
@@ -59,10 +75,22 @@ export const ClusterInfo = ({clusterTitle, additionalClusterInfo = []}: ClusterI
 
     const fetchData = useCallback(() => {
         dispatch(getClusterInfo(clusterName ? String(clusterName) : undefined));
+        dispatch(getClusterNodes());
     }, [dispatch, clusterName]);
 
     useAutofetcher(fetchData, [fetchData], true);
 
+    const versionToColor = useMemo(() => {
+        if (additionalVersionsProps?.getVersionToColorMap) {
+            return additionalVersionsProps?.getVersionToColorMap();
+        }
+        return parseVersionsToVersionToColorMap(cluster?.Versions);
+    }, [additionalVersionsProps, cluster]);
+
+    const versionsValues = useMemo(() => {
+        return parseNodesToVersionsValues(nodes, versionToColor);
```

Comment:
As we don't have nodes count for specific versions in `viewer/json/cluster` yet, we calculate it